static/js/editor.ts

```diff
@@ -0,0 +1,106 @@
+export type Breakpoints = Record<number, string>;
+
+
+export interface HedyEditorCreator {
```

Comment:
Ahh, I see what you're doing. Makes it easy to switch out all different entry points in one go later, right? 

Clever!